website/content/en/preview/faq.md

```diff
@@ -133,3 +133,24 @@ See [Deprovisioning nodes]({{< ref "./tasks/deprovisioning" >}}) for information
 ### How do I upgrade Karpenter?
 Karpenter is a controller that runs in your cluster, but it is not tied to a specific Kubernetes version, as the Cluster Autoscaler is.
 Use your existing upgrade mechanisms to upgrade your core add-ons in Kubernetes and keep Karpenter up to date on bug fixes and new features.
+
+### How to upgrade from previous version (before v0.6.1) to v0.6.2
```

Comment:
I'm not sure we want this as an FAQ. Can you add in the release notes instead?

Reply:
Don't we want to keep it somewhere permanent? We will need this for all the version after v0.6.2.

Reply:
We will need these instructions for the future versions as well for users who skipped v0.6.2.  I think we should keep it somewhere permanent if not the FAQ.